packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import '../material/material.dart'; // non ideal
```

Comment:
We shouldn't import this library.  Cupertino isn't allowed to depend on Material.